test/orchard/info_test.clj

```diff
@@ -368,7 +369,7 @@
                     (map #(select-keys % [:ns :name :arglists :macro :file]))))))
 
       (testing "- :clj"
-        (is (= (take 2 (repeat expected))
+        (is (= [{}, expected]
```

Comment:
Changed this expectation, `orchard.test-ns/my-add` does not exist so the right result for it is `{}`